projects/app-lob/src/app/grid-finjs/controllers.component.ts

```diff
@@ -60,6 +65,11 @@ export class ControllerComponent implements OnInit, OnDestroy {
             .subscribe(() => this.frequencyChanged.emit(this.intervalSlider.value as number));
         
         this.toast.positionSettings.verticalDirection = VerticalAlignment.Middle;
+
+        // Hide theme switcher
+        if (this.router.snapshot.queryParamMap.get('theme-switch')) {

```

Comment:
It seems it does not matter what value is send for the theme-switch param.Even if ?theme-switch=true is send, button will be hidden, Let's respect the value sent.